zerver/tests/test_signup.py

```diff
@@ -406,8 +406,8 @@ def test_multi_user_invite(self):
         self.login("hamlet@zulip.com")
         # Intentionally use a weird string.
         self.assert_json_success(self.invite(
-"""bob-test@zulip.com,     carol-test@zulip.com,
-dave-test@zulip.com
+            """bob-test@zulip.com,     carol-test@zulip.com,
+            dave-test@zulip.com
```

Comment:
I think this will not produce the same output as before, since whitespace inside a `"""` block is actually used at runtime....